taf/tests/test_updater/update_utils.py

```diff
@@ -256,3 +296,31 @@ def _update_expect_error():
                 assert client_repository.path.exists()
             else:
                 assert not client_repository.path.exists()
+
+
+def invalidate_auth_repo(auth_repo):
```

Comment:
Let's move this to conftest, I'd like all functions which update the repositories to be in the same place.